src/Classes/Models/ApplicantModel.php

```diff
@@ -201,4 +203,11 @@ public function createNewApplicant(
             $applicantNotes
         );
     }
+
+    public function deleteApplicant($id)
+    {
+        $query= $this->dbConnection->prepare( "UPDATE `applicants` SET `deleted` = '1' WHERE `id` = :id");
```

Comment:
space after query :)

Reply:
made spaces around = consistent with other functions in file.